Orange/widgets/evaluate/owpredictions.py

```diff
@@ -442,15 +461,15 @@ def _commit_predictions(self):
             self.send("Predictions", None)
             return
 
-        class_var = self.class_var
-        if class_var and class_var.is_discrete:
+        if self.class_var and self.class_var.is_discrete:
             newmetas, newcolumns = self._classification_output_columns()
         else:
             newmetas, newcolumns = self._regression_output_columns()
 
         attrs = list(self.data.domain.attributes) if self.output_attrs else []
         metas = list(self.data.domain.metas) + newmetas
-        domain = Orange.data.Domain(attrs, class_var, metas=metas)
+        domain = \
+            Orange.data.Domain(attrs, self.data.domain.class_var, metas=metas)
```

Comment:
Now I recalled: I did have `self.class_var` and it fit into a single line. But this fails exactly in the case that this PR is fixing: if `self.data.domain.class_var` is `None`, the output data should have no class, either. Changing this to `self.class_var` can add a column of unknown values if the original data has no class.